L/LibUCL/build_tarballs.jl

```diff
@@ -0,0 +1,42 @@
+using BinaryBuilder
+
+name = "LibUCL"
+version = v"0.8.2"
+
+sources = [GitSource("https://github.com/vstakhov/libucl.git",
+                     "a615938cec3ae35c70aa2fb9845c9c12e5c7326f")]
+
+script = raw"""
+cd ${WORKSPACE}/srcdir/libucl*
+if [[ "${target}" == *-mingw* ]]; then
+    cp "${includedir}/pcreposix.h" "${includedir}/regex.h"
+    sed -i 's/-lregex/-lpcreposix-0/' configure
```

Comment:
Actually this probably needs to be `configure.ac`, or maybe just `./autogen.sh` should come first

Reply:
Ugh, you're right

Reply:
Actually if we're using PCRE, this probably needs to be
```suggestion
    sed -i 's/-lregex/-lpcre/' configure
```
right? Or does the PCRE builder include a `libpcreposix-0.dll` and it just isn't listed as a library product in the build recipe?

Reply:
I'm away from computer now, can't check.

Reply:
> Or does the PCRE builder include a libpcreposix-0.dll and it just isn't listed as a library product in the build recipe?

That's indeed the case.